sdk/search/azure-search-documents/src/test/java/com/azure/search/documents/indexes/IndexersManagementSyncTests.java

```diff
@@ -190,7 +188,7 @@ public void canCreateAndListIndexerNames() {
         searchIndexerClient.createIndexer(indexer2);
         indexersToDelete.add(indexer2.getName());
 
-        Iterator<String> indexersRes = searchIndexerClient.listIndexerNames(generateRequestOptions(), Context.NONE)
+        Iterator<String> indexersRes = searchIndexerClient.listIndexerNames(Context.NONE)
```

Comment:
Could you have a test on setting request id?

Reply:
Will add a few tests. As a heads up we never validated the `x-ms-client-request-id` returned from the service match what was set in `RequestOptions` so this will be even more important to do.